b/behat_minkextension/behat_minkextension_v2.3_v2.3.1_rhel_8.4.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : MinkExtension
+# Version       : v2.3, v2.3.1
```

Comment:
As per line 18, build script is created for version v2.3. Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.